mathics/builtin/system.py

```diff
@@ -882,12 +882,12 @@ class Breakpoint(Builtin):
            variable `PYTHONBREAKPOINT` can be utilized.
     </dl>
 
+    >> (* Test with a dummy breakpoint. *)
```

Comment:
I've been thinking about this, and even outside of testing it might be useful to have a _disabled_  as opposed to "dummy" breakpoint. 

Should we decide this,  I guess this should not be put in docpipeline, since it would no longer be about documentation. 

I am curious what other folks think and suggest.

Reply:
From my point of view, if a `disabled_breakpoint` function outside of doctests would be useful, I'll gladly change the current implementation and move the function to `main`.

Reply:
Waiting on a final decision on this.

Reply:
> From my point of view, if a `disabled_breakpoint` function outside of doctests would be useful, I'll gladly change the current implementation and move the function to `main`.

If we go that route (and I am for it), then it might be slightly simpler to put this inside `mathics/__init__.py`. So it can be referred to as `mathics.disabled_breakpoint`. 

Normally putting functions inside `__init__.py` is considered a bad thing, especially when there is a lot of code and initialization code. This is because it the order in which code runs can be indeterminate or hard to figure out  and may change as imports change. (In my view this module system for Python is a misfeature that has persisted for too long). 

However in this particular situation where we have a short function  and that does not do any initialization, this kind of things is okay since it simplifies the user interface.

Reply:
> Waiting on a final decision on this.

@mmatera ?

Reply:
If it is possible to handle it by defining a breakpoint handler, it would be great! About the proper place to leave it, I am OK about put it in `mathics__init__.py`, or whatever other place that makes sense to you.